### DISCUSSION TITLE
Task 3 - LAB 02.04

### DISCUSSION THREAD

**Discussion by Andresfga** [rramosp/ai4eng.v1#82] [02 - PYTHON]
Buenas tardes.
Logro hacer que el programa me determine la ultima posición de la carta escogida en las dos opciones que están en el LAB para ensayarlo, pero momento de registrar la nota me sale este error y no se que podría ser lo que esta fallando, me gustaría saber si me pueden ayudar con esto, muchas gracias.

![image](https://user-images.githubusercontent.com/95330160/145722396-6f5ce86a-8993-42be-b221-bc530c78c246.png)

**Comment by Felipe-RA** [comment]
Saludos Andresfga

No puedo ver el error exacto, despliegalo completamente para poder identificar el problema

También tu código
